staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -223,6 +226,9 @@ type Config struct {
 	// EquivalentResourceRegistry provides information about resources equivalent to a given resource,
 	// and the kind associated with a given resource. As resources are installed, they are registered here.
 	EquivalentResourceRegistry runtime.EquivalentResourceRegistry
+
+	// APIServerID is the ID of this API server
+	APIServerID string
```

Comment:
As an administrator, I'd like to know my apiserver's current id by e.g. visiting some page on apiserver. Can I do that? What do you think about adding a standalone metric, e.g. where apiserver-id is a label? I'm totally open to other ideas too.

Reply:
Good point. I was relying on the hostname+pid to provide the debug info. 

Having it in a metric makes sense. A similar case is the `/version` endpoint, which also has a metric "kubernetes_build_info" mapped in `/metrics`. We can start with a standalone metric. 

What do you think of having this in a followup PR? I think it can be a Beta graduate criteria, if the name / format gets controversial.

Reply:
Sure, a followup sounds fine.